local/e2e/cli-only/e2e_test.go

```diff
@@ -203,10 +203,10 @@ func TestContextMetrics(t *testing.T) {
 		assert.DeepEqual(t, []string{
 			`{"command":"context create","context":"moby","source":"cli","status":"success"}`,
 			`{"command":"ps","context":"moby","source":"cli","status":"success"}`,
-			`{"command":"context use","context":"moby","source":"cli","status":"success"}`,
+			`{"command":"context use","context":"local","source":"cli","status":"success"}`,
 			`{"command":"ps","context":"local","source":"cli","status":"success"}`,
 			`{"command":"stop","context":"local","source":"cli","status":"failure"}`,
-			`{"command":"context use","context":"local","source":"cli","status":"success"}`,
+			`{"command":"context use","context":"moby","source":"cli","status":"success"}`,
```

Comment:
this changed because now, we capture the context AFTER the command ran instead of before. i'm not sure if this matters.